test/framework/github.py

```diff
@@ -1317,20 +1318,37 @@ def test_github_create_test_report(self):
                 'log_file': logfile,
             }),
         ]
+        test_remove_name = 'REMOVEME'
+        test_remove_val = 'test-abc'
```

Comment:
Maybe make this another dict and add a few different casings since the matching is loose with respect to case?

Reply:
Improved with https://github.com/easybuilders/easybuild-framework/pull/4877/commits/7403a42766de1124cd96edc71c6ac1f416378ad7